packages/http-caching-proxy/src/__tests__/integration/http-caching-proxy.integration.ts

```diff
@@ -62,28 +82,26 @@ describe('HttpCachingProxy', () => {
     await givenRunningProxy({logError: false});
     await expect(
       makeRequest({
-        uri: 'http://does-not-exist.example.com',
-        proxy: proxy.url,
-        resolveWithFullResponse: true,
+        url: 'http://does-not-exist.example.com',
+        proxy: getProxyConfig(),
       }),
     ).to.be.rejectedWith(
       // The error can be
       // '502 - "Error: getaddrinfo EAI_AGAIN does-not-exist.example.com:80"'
       // '502 - "Error: getaddrinfo ENOTFOUND does-not-exist.example.com'
-      /502 - "Error\: getaddrinfo/,
+      /502 - getaddrinfo/,
```

Comment:
Please update the example errors in the code comment above to match the real error messages.